src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -263,10 +266,16 @@ def upgrade():
         nullable=True,
     )
     op.add_column(
-        "privacynoticehistory", sa.Column("language", sa.String(), nullable=False)
+        "privacynoticehistory",
+        sa.Column(
+            "language", sa.String(), server_default="en", nullable=False
+        ),  # set "en" as a server default for existing values ... TODO: maybe make temporarily nullable and set english 'manually' instead ??
     )
     op.add_column(
-        "privacynoticehistory", sa.Column("title", sa.String(), nullable=False)
+        "privacynoticehistory",
+        sa.Column(
```

Comment:
i think with this one we can set nullable temporarily here and then set to non-nullable on our post-data-migration schema migration (which will also be removing unneeded columns), right? i.e. the values here should be populated by the data migration, for every existing row, i think...

Reply:
I agree, nullable temporarily would be good, and data migration can populate.

Reply:
makes sense, thanks for pointing this out on the data migration side too!